google/cloud/pubsub_v1/publisher/client.py

```diff
@@ -272,7 +272,13 @@ def resume_publish(self, topic, ordering_key):
                 sequencer.unpause()
 
     def publish(
-        self, topic, data, ordering_key="", retry=gapic_v1.method.DEFAULT, **attrs
+        self,
+        topic,
+        data,
+        ordering_key="",
+        retry=gapic_v1.method.DEFAULT,
```

Comment:
If retry and timeout are added to PublisherOptions, then there's no need to add these parameters here, yes?

Reply:
Somebody might want to override these default options for a few particular `publish()` calls?

I'd say we keep these two arguments for flexibility, as well as for consistency with other Publisher methods. Besides, the `publish()` method signature already includes the `retry` parameter and thus shouldn't be removed, and it makes more sense to retain `timeout` with it.